server/routes/user.js

```diff
@@ -0,0 +1,34 @@
+const userRouter = require("express").Router();
```

Comment:
in general, add protections against sql injection, and add protected routes if only accessible to users/admin